.gitlab-ci.yml

```diff
@@ -131,12 +131,12 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v6041145-dc43691
-  DATADOG_AGENT_BUILDERS: v5641696-27f2a62
-  DATADOG_AGENT_WINBUILDIMAGES: v6420769-18bf4e9
-  DATADOG_AGENT_ARMBUILDIMAGES: v5782992-59363bf
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v5782992-59363bf
-  DATADOG_AGENT_NIKOS_BUILDIMAGES: v5782992-59363bf
+  DATADOG_AGENT_BUILDIMAGES: v6496660-25ba5f7
+  DATADOG_AGENT_BUILDERS: v6496182-a83d86c
+  DATADOG_AGENT_WINBUILDIMAGES: v6496660-25ba5f7
+  DATADOG_AGENT_ARMBUILDIMAGES: v6496660-25ba5f7
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v6496660-25ba5f7
+  DATADOG_AGENT_NIKOS_BUILDIMAGES: v6496660-25ba5f7
```

Comment:
Once the related `datadog-agent-buildimages` and `datadog-agent-builders` PRs are merged, please change these versions to ones from the resulting `main` pipelines and check the resulting `datadog-agent` pipeline before merging.